sdk/eventhub/azure-eventhub/tests/perfstress_tests/T1_legacy_tests/send_event_batch.py

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _SendTest
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from azure.eventhub import EventData
+
+
+class LegacySendEventBatchTest(_SendTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.data = get_random_bytes(self.args.event_size)
+
+    def data_generator(self):
+        for i in range(self.args.num_events):
+            yield self.data
+
+    def run_sync(self):
+        event = EventData(batch=self.data_generator())
```

Comment:
qq: Why did you use this line instead of importing and sending `EventDataBatch`? (same on line 27)

Reply:
haha, I'm glad you asked this -- because there's no EventDataBatch in track1 and this is the way we send batch in track1 :P

(code here: https://github.com/Azure/azure-sdk-for-python/blob/release/eventhub-v1/sdk/eventhub/azure-eventhubs/azure/eventhub/common.py)

Reply:
oh 🤦 lol